devtools/server/actors/replay/module.js

```diff
@@ -273,6 +275,18 @@ function setSourceMap({
   });
 }
 
+function checkRecordingCreated() {
+  if (RecordReplayControl.isRecordingCreated()) {
```

Comment:
Given the semantics of RecordReplayIsRecordingCreated this interval will never finish when the recording is being written to disk, right?  It seems nice if we could avoid that overhead, maybe the API should return true if the recording was written to disk (which seems to fit with the intuitive view of the API).

Reply:
I addressed that with https://github.com/replayio/gecko-dev/pull/901/files#diff-6a6c1addab069131d9a93e08d30c2902137cf354ab906e8f09346e6d0712dc40R772 but I agree that the semantics aren't the best. I thought about inverting the function name to be like `isRecordingCreationPending` or something but that's definitely more verbose.

If you're happy with it returning `true` when not uploading, I'm happy to do that though.

Reply:
We could always assert in the function instead, as another alternative. I also considered that and would be happy to do so, it just makes it less consistent with `WaitForRecordingCreated`

Reply:
Since WaitForRecordingCreated immediately returns if the recording is being written to disk, it seems like the most consistent semantics are for IsRecordingCreated to return true if that is the case as well.  I didn't notice the special case in IsRecordingCreated(), I think it's nicer when functions like this are simple wrappers for the recorder API.